presto-orc/src/main/java/io/prestosql/orc/writer/TimestampColumnWriter.java

```diff
@@ -316,15 +318,6 @@ public void reset()
         statisticsBuilder = statisticsBuilderSupplier.get();
     }
 
-    private void writeTimestampMillis(Block block)
```

Comment:
Good catch on the unused method, but please remove in a separate commit, since it's unrelated to adding precision support.